docs/configuration.md

```
@@ -302,6 +310,30 @@ ignore_exceed: false          # Applies to takt run and takt watch like --ignore
 #         model: ollama-cloud/gemma4:31b
 ```
 
+#### OpenCode execution guards
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)